### PR TITLE
feat(website): fonts, colors, images to be more IDE like

### PR DESCRIPTION
**Pull request by jayagl**
Made some interim updates to the website to make it look a bit more themed:

1. Updated font to monospace 
2. Updated background, primary and secondary colors
3. Made code blocks on index page look consistent with those on docs
4. Updated feature images - I realized they were the docusaurus stock images, so I generated some silly ones using dalle2 lol 🤖. I hit my daily limit of requests so wasn't able to get what I wanted, i.e. some kind of two tone pixel art. Feel free to exclude them or suggest different imagery to generate later today

Screen Shot 2022-06-29 at 9 51 33 AM

Screen Shot 2022-06-29 at 9 26 10 AM

### CODE REVIEW

File: website/src/components/HomepageFeatures/index.tsx [hunk 1]
```diff
@@ -108,14 +110,24 @@ function CodePreview(props: { title: string; code: string }) {
...
+          {tokens.map((line, i) => (
+            <div {...getLineProps({ line, key: i })}>
+              {line.map((token, key) => (
+                <span {...getTokenProps({ token, key })} />
+              ))}
+            </div>
+          ))}
+        </pre>
+      )}
+    </Highlight>
```
Comment (thantos) [inline website/src/components/HomepageFeatures/index.tsx:130]: There is a functionless highlighter component that includes all of this and correctly highlights Function

Comment (thantos) [reply website/src/components/HomepageFeatures/index.tsx:130]: https://github.com/functionless/functionless/blob/main/website/src/components/highlighter.tsx#L22

File: website/src/css/custom.css [hunk 2]
```diff
@@ -4,43 +4,34 @@
...
-  --ifm-color-primary: #24cc85;
-  --ifm-color-primary-dark: #20b878;
-  --ifm-color-primary-darker: #1fad71;
-  --ifm-color-primary-darkest: #198f5d;
-}
+/* For readability concerns, you should choose a lighter palette in dark mode. */
+[data-theme="dark"],
+:root[data-theme="dark"],
+html[data-theme="dark"] {
+  /* lazily setting all colors to be the same, can't see the shades being used anywhere rn*/
```
Comment (thantos) [inline website/src/css/custom.css:11]: this is where the shades came from: https://docusaurus.io/docs/styling-layout#styling-your-site-with-infima

Also not sure how they are used.

**Review by thantos — CHANGES_REQUESTED** [review]

**Comment by sam-goodwin** [comment]
Thank you for the change @jayagl! Our design needs so much work, every little bit helps.

**Comment by jayagl** [comment]
Purple is from the [vsDark](https://github.com/FormidableLabs/prism-react-renderer/tree/master/src/themes) theme. Any suggestions for which theme might go better?

**Comment by jayagl** [comment]
Went with [this one](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/themes/okaidia.js), looks good imo

File: website/src/css/custom.css (hunk 2 above)
Comment (jayagl) [reply website/src/css/custom.css:11]: Added in the colors in case they are useful

**Review by sam-goodwin — APPROVED** [review]

**Review by thantos — APPROVED** [review]

**Pull request merged**